### PR TITLE
Add notes for several suggestions

### PR DESCRIPTION
**Pull request by scottgonzalez**

### CODE REVIEW

**Comment by wooorm** [comment]
@scottgonzalez Ping!

**Comment by scottgonzalez** [comment]
Sorry, I had to replace my laptop, so I haven't had much availability. I will get back to this though.

File: script/ablist.yml [hunk 1]
```diff
@@ -11,7 +12,7 @@
     - disabled
 - type: simple
   source: http://ncdj.org/style-guide/
-  note: If possible, describe exacly what this is.
+  note: If possible, describe the specific birth defect.
```
Comment (Murderlon) [inline script/ablist.yml]: @scottgonzalez disabilities don't have to necessarily come from birth defects, right?

Comment (scottgonzalez) [reply script/ablist.yml]: That's correct, but the only term that will trigger this note (the only `inconsiderate` value) is "birth defect."

**Comment by Murderlon** [comment]
@scottgonzalez the changes looks good, would mind fixing this test so we can merge? :)
https://github.com/retextjs/retext-equality/blob/413dac5b2b/test.js#L14-L20

**Comment by scottgonzalez** [comment]
updated

**Comment by Murderlon** [comment]
@scottgonzalez thank you!

**Pull request merged**